sphinxserve/__init__.py

```diff
@@ -48,6 +52,13 @@
                         short: d
                         action: store_true
                         default: __SUPPRESS__
```

Comment:
Oh -- are you going to be at Pycon?  I'll be there too; maybe I'll see you there!